packages/graphback-runtime/src/resolvers/LayeredRuntimeResolverGen.ts

```diff
@@ -91,12 +91,12 @@ export class LayeredRuntimeResolverGenerator {
   }
 
   // tslint:disable-next-line: no-any
-  private createSubscriptions(resolverElement: InputModelTypeContext, resolvers: any) {
+  private createSubscriptions(resolverElement: InputModelTypeContext, resolvers: any, objectName: string) {
```

Comment:
There is no longer a need to pass the entire `resolverElement` anymore if you pass `config`

```suggestion
private createSubscriptions(config: GraphbackGeneratorConfig, resolvers: any, objectName: string) {
```